subnetGroups/subnet-groups.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "subnetGroupName": "Defi Kingdoms",
+    "explorerUri": "defi-kingdoms",
+    "mainnetSubnetIds": ["Vn3aX6hNRstj5VHHm63TCgPNaeGnRSqCYXQqemSqDd2TQH4qJ"],
+    "testnetSubnetIds": ["XHLRR9cvMtCR8KZsjU8nLxg1JbV7aS23AcLVeBMVHLKkSBriS"]
+  },
+  {
+    "subnetGroupName": "Swimmer Network",
+    "explorerUri": "swimmer",
+    "mainnetSubnetIds": ["2e3CqLY8thW1WKS89WQ7WR9Qy2bCrrP5mv7psqMRGCHGXuQ1Uj"],
```

Comment:
Yep, that's why I made both `mainnetSubnetIds` and `testnetSubnetIds` an array, so that we can easily extend them later.  Did you have something else in mind?